rerun_cpp/src/rerun/components/line_strip2d_ext.cpp

```diff
@@ -0,0 +1,23 @@
+#include "line_strip2d.hpp"
+
+#include <algorithm>
+
+// #define EDIT_EXTENSION
+
+namespace rerun {
+    namespace components {
+
+#ifdef EDIT_EXTENSION
+        // [CODEGEN COPY TO HEADER START]
+
+        template <typename T>
+        LineStrip2D(const std::vector<T>& points_) : points(points_.size()) {
+            std::transform(points_.begin(), points_.end(), points.begin(), [](const T& pt) {
+                return rerun::datatypes::Vec2D(pt);
+            });
+        }
```

Comment:
I think we might be able to get rid of this in the very near future by having `LineStrip2D` hold a `rerun::Collection` 🤔